### PR TITLE
Wqp 799

### PR DESCRIPTION
**Pull request by ayan-usgs**
Three new templates and a lot of .pngs.

### CODE REVIEW

File: wqp/wqx/templates/wqx/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<html>
+<head>
+	<title>WQX-Outbound Schema Info</title>	<style type="text/css" media="screen">		.instructions {			text-align: center;			border-width: 2pt;			border-color: #333333;			border-style: solid;			background-color: #FFEE99;			padding-top: 4pt;			padding-bottom: 2pt;			padding-left: 8pt;			padding-rightt: 8pt;			margin-bottom: 8pt;		}		.note {			font-style: italic;			font-size: small;		}		body {			font-family: arial;		}	</style>
```
Comment (mbucknell) [inline wqp/wqx/templates/wqx/index.html]: This seems a little wonky. I don't see the ending </title> tag or the beginning <script> tag.

File: wqp/wqx/templates/wqx/index.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+</head>
+<body>	<div class="instructions">The National Water Information System (NWIS) Water-Quality 	Web Services conform to the format defined in the below referenced XML schema.	</div>	<b>WQX-Outbound XML schema information:</b>
```
Comment (mbucknell) [inline wqp/wqx/templates/wqx/index.html]: This also seems wonky with the multiple spaces between "Water-Quality" and "Web Services"

Comment (ayan-usgs) [inline wqp/wqx/templates/wqx/index.html]: That's what I get for using the existing index.html verbatim. :gun:

The weird tab has been removed.

File: wqp/wqx/templates/wqx/index.html (hunk 1 above)
Comment (ayan-usgs) [inline wqp/wqx/templates/wqx/index.html]: Which html tag isn't present?

Comment (mbucknell) [inline wqp/wqx/templates/wqx/index.html]: They are all there. My mistake but I don't like putting the style on the same line as the title tag. It's just formatting but it makes it hard to read.

**Pull request merged**